lib/Github/Exception/ApiLimitExceedException.php

```diff
@@ -9,9 +9,17 @@
  */
 class ApiLimitExceedException extends RuntimeException
 {
+    /** @var int */
     private $limit;
+    /** @var int */
     private $reset;
 
+    /**
+     * @param int             $limit
+     * @param int             $reset
+     * @param int             $code
+     * @param \Throwable|null $previous
```

Comment:
Doesn't 2.x support PHP 5 still? Is previous actually always an Exception? Can we just typehint `\Exception|null` here?

Reply:
No, I've bumped php a while ago. We support 7.1+

https://github.com/KnpLabs/php-github-api/blob/6a80368db3d9f742df0c109a27a2d9227b1800d5/composer.json#L20